Cargo.toml

```diff
@@ -13,18 +13,18 @@ categories = ["web-programming"]
 
 [dependencies]
 tracing = { version = "0.1.40" }
-opentelemetry = { version = "0.24.0" }
-opentelemetry_sdk = { version = "0.24.1", features = ["rt-tokio"] }
-opentelemetry-application-insights = { version = "0.34.0", features = ["reqwest-client", "metrics", "live-metrics"] }
-tracing-opentelemetry = { version = "0.25.0" }
-tracing-subscriber = { version = "0.3.17" }
-futures = { version = "0.3.28" }
-http = { version = "1.1.0" }
+opentelemetry = { version = "0.27.1" }
+opentelemetry_sdk = { version = "0.27.1", features = ["rt-tokio"] }
+opentelemetry-application-insights = { version = "0.37.0", features = ["reqwest-client", "metrics", "live-metrics"] }
+tracing-opentelemetry = { version = "0.28.0" }
+tracing-subscriber = { version = "0.3" }
+futures = { version = "0.3" }
+http = { version = "1.2.0" }
 http-body-util = { version = "0.1.2" }
-axum = { version = "0.7.5", features = ["macros"] }
-hyper = { version = "1.4.1", features = ["full"] }
-tower = { version = "0.4.13", features = ["full"] }
-serde = { version = "1.0.126" }
-serde_json = { version = "1.0.64" }
-tokio = { version = "1.29.1", features = ["rt", "macros", "signal", "parking_lot"] }
-reqwest = { version = "0.12.5", features = ["blocking"] }
\ No newline at end of file
+axum = { version = "0.8.1", features = ["macros"] }
+hyper = { version = "1.5.2", features = ["full"] }
+tower = { version = "0.5.2", features = ["full"] }
+serde = { version = "1" }
```

Comment:
I changed tokio & serde version pins to major version, they should be compatible between major versions